ReactiveCocoaTests/InterceptingSpec.swift

```diff
@@ -830,36 +834,28 @@ private class ForwardInvocationTestObject: InterceptedObject {
 	var forwardedCount = 0
 	var forwardedSelector: Selector?
 
-	override open class func initialize() {
-		struct Static {
-			static var token: Int = {
-				let impl: @convention(c) (Any, Selector, AnyObject) -> Void = { object, _, invocation in
-					let object = object as! ForwardInvocationTestObject
-					object.forwardedCount += 1
-					object.forwardedSelector = invocation.selector
-				}
-
-				let success = class_addMethod(ForwardInvocationTestObject.self,
-				                ObjCSelector.forwardInvocation,
-				                unsafeBitCast(impl, to: IMP.self),
-				                ObjCMethodEncoding.forwardInvocation)
-
-				assert(success)
-				assert(ForwardInvocationTestObject.instancesRespond(to: ObjCSelector.forwardInvocation))
+	static func _initialize() {
+		let impl: @convention(c) (Any, Selector, AnyObject) -> Void = { object, _, invocation in
+			let object = object as! ForwardInvocationTestObject
+			object.forwardedCount += 1
+			object.forwardedSelector = invocation.selector
+		}
 
-				let success2 = class_addMethod(ForwardInvocationTestObject.self,
-				                               ForwardInvocationTestObject.forwardedSelector,
-				                               _rac_objc_msgForward,
-				                               ObjCMethodEncoding.forwardInvocation)
+		let success = class_addMethod(ForwardInvocationTestObject.self,
+		                              ObjCSelector.forwardInvocation,
+		                              unsafeBitCast(impl, to: IMP.self),
+		                              ObjCMethodEncoding.forwardInvocation)
 
-				assert(success2)
-				assert(ForwardInvocationTestObject.instancesRespond(to: ForwardInvocationTestObject.forwardedSelector))
+		assert(success)
+		assert(ForwardInvocationTestObject.instancesRespond(to: ObjCSelector.forwardInvocation))
 
-				return 0
-			}()
-		}
+		let success2 = class_addMethod(ForwardInvocationTestObject.self,
+		                               ForwardInvocationTestObject.forwardedSelector,
+		                               _rac_objc_msgForward,
+		                               ObjCMethodEncoding.forwardInvocation)
 
-		_ = Static.token
```

Comment:
> it is not for production 

Don't we run tests in release mode though? We used to, and if not I'd strongly recommend it cause it's how we find Swift bugs with `-O`. And then we wouldn't want `assert` getting ignored here.

> they are no expected to fail

Even more reason to make them preconditions then.